django_project/localities_osm/models/locality.py

```diff
@@ -0,0 +1,85 @@
+__author__ = 'Irwan Fathurrahman <irwan@kartoza.com>'
+__date__ = '07/01/19'
+
+from django.contrib.gis.db import models
+from localities_osm.models.base import LocalityOSMBase
+from localities_osm.querysets import (
+    PassThroughGeoManager,
+    OSMQuerySet
+)
+
+
+class LocalityOSM(LocalityOSMBase):
+    osm_id = models.BigIntegerField()
+    type = models.CharField(
+        max_length=512, blank=True, null=True)
+    name = models.CharField(
+        max_length=512, blank=True, null=True)
+    emergency = models.CharField(
+        max_length=512, blank=True, null=True)
+    operator = models.CharField(
+        max_length=512, blank=True, null=True)
+    opening_hours = models.CharField(
+        max_length=512, blank=True, null=True)
+    contact_website = models.CharField(
+        max_length=512, blank=True, null=True)
+    contact_phone = models.CharField(
+        max_length=512, blank=True, null=True)
+    phone = models.CharField(
+        max_length=512, blank=True, null=True)
+    website = models.CharField(
+        max_length=512, blank=True, null=True)
+    speciality = models.CharField(
+        max_length=512, blank=True, null=True)
+
```

Comment:
We could have 8 more attributes mapped in here from OSM - see https://github.com/hotosm/health/wiki/Health-facility-tagging-within-OSM-(proposal) and https://github.com/healthsites/healthsites/wiki/Healthsite-attributes